### PR TITLE
Ltsva fix

### PR DESCRIPTION
**Pull request by jwbishop**
Changes ltsva output argument order and fixes typographic bug. This was a prior patch that was reverted and never re-merged.

### CODE REVIEW

File: lts_array/ltsva.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def ltsva(st, rij, WINLEN, WINOVER, ALPHA=1.0):
         if len(stns) > 0:
             tval = str(t[jj])
             stdict[tval] = stns
-        if jj == (nits-1) and alpha != 1.0:
+        if jj == (nits-1) and ALPHA != 1.0:
```
Comment (liamtoney) [inline lts_array/ltsva.py]: Why is this changing to all caps?

**Comment by liamtoney** [comment]
The input args for `ltsva()` should be lowercase.

**Review by liamtoney — CHANGES_REQUESTED** [review]
The input args for `ltsva()` should be lowercase.

File: lts_array/ltsva.py (hunk 1 above)
Comment (jwbishop) [reply lts_array/ltsva.py]: It was initially to bring the code to consistency. But now everything in the function has been downcast.

**Review by liamtoney — APPROVED** [review]
Looks good!

**Review by davidfee5 — APPROVED** [review]
Looks good to me!

**Pull request merged**